cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -199,6 +199,8 @@ function _add_common_params() {
 
     if [ "$KUBEVIRT_DEPLOY_CDI" == "true" ]; then
         params=" --deploy-cdi $params"
+    else
+        params=" --deploy-cdi=false $params"
```

Comment:
This shouldn't be necessary - iiuc the default for cdi deployment is flipped inside kubevirt/kubevirt: https://github.com/kubevirt/kubevirt/blob/0e54154a8a95745aa7cd54ccd03597c37b1269ed/hack/cluster-up.sh#L20

Reply:
Yes, with changes from this [commit](https://github.com/kubevirt/kubevirtci/commit/22e4513dd963abd45332f2972e3021423b699d6d#diff-12e7b6e3d4ce92e7d50b6cd0e3aebac95e9480cb4dbbe4a220e1c86c3b12f6ebL124), I don't need to handle this here anymore. 
P.S: Earlier when discussing with Ammar, he suggested to change it here than changing the default in the run.go as that was the default in the shell scripts before he migrated.

Thanks for pointing this.